### PR TITLE
Use link name when title is blank

### PR DESCRIPTION
**Pull request by kylecesmat**
Sets title to text rather than null when title is not provided

### CODE REVIEW

**Comment by kylecesmat** [comment]
cc @paulathevalley

File: src/markdown-renderers/append-link-icon.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default {
     const Icon = href.includes("formidable.com") ? IconInternalLink : IconExternalLink;
 
     return (
-      `<a href=${href} title=${title}>
+      `<a href=${href} title=${title || text}>
```
Comment (paulathevalley) [inline src/markdown-renderers/append-link-icon.js]: If `title` is blank, then maybe it should be an empty string `""`, similar to `alt=""` for ``.

Comment (kylecesmat) [inline src/markdown-renderers/append-link-icon.js]: Ah - you're probably correct in that. Better default approach.

**Comment by paulathevalley** [comment]
:+1:

**Pull request merged**